### ISSUE TITLE
keep trying clock set on network fail

### ISSUE THREAD

**Issue by kantarjiev** [Phambili-Tech/Newt_RV_3028_C7#63]
I've had a bunch of power outages at home lately. It often seems to be the case that my Newt powers up well before the Wifi and/or Internet return. This means that the time doesn't get set, and sadly it remains unset. 

Maybe this is the loop of death, or related - unsure. It doesn't seem to be looping.

It would be nice if the Newt would, in this case where it must know that it didn't get a time set packet, retry every 10 minutes or so until it gets somewhere...

**Comment by darianbjohnson** [comment]
Yeah, it's not related to "loop of death". Newt sets the clock on the initial boot up... but if it doesn't set the time, it never tries again. This is a bug. I need to add "time sync" retry logic
